dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/Engine.scala

```diff
@@ -116,11 +116,11 @@ class Engine(context: EngineContext) {
       }
     }
 
-    tasks.foreach(task => submitTask(task, sources))
+    tasks.par.foreach(task => submitTask(task, sources))
```

Comment:
On this PR, the `CryptographyMisuseTests` become flaky. I would think it's this line. Verifying...

Reply:
I wonder what the expense is of submitting tasks in serial is, but yes this would add non-determinism to the execution order.

Reply:
@DavidBakerEffendi Post initial task submission, we merge the results to the final results table and also submit partial tasks returned as a result of the initial task submission. Merging of results is a little time taking as compared to task submission.

Reply:
This is done.